### PR TITLE
A full info query now, using a fragment

### PR DESCRIPTION
**Pull request by narration-sd**
to dig out both language and files in one go. Simple formatting suits this as a test-only page, while showing all. The folder.contents is necessary because of the nesting of the original GitApi schema.

### CODE REVIEW

File: src/pages/JoseConnect.vue [hunk 1]
```diff
@@ -22,69 +38,79 @@
...
     }
+  },
+  mounted () {
+    // console.log ('results: ' + JSON.stringify(this.$page))
+    console.log ('results: ' +
+      JSON.stringify(this.$page.gitapi.repos.repositories.nodes[4].docs.folders[0].lang))
+    console.log ('results: ' +
+      JSON.stringify(this.$page.gitapi.repos.repositories.nodes[4].docs.folders[0].folder.files[0]))
   }
 }
```
Comment (jurra) [inline src/pages/JoseConnect.vue]: I cloned your fork, tested this and it throws the following error: 
```
vue.runtime.esm.js?2b0e:619 [Vue warn]: Error in mounted hook: "TypeError: Cannot read property 'repositories' of undefined"
```

**Comment by jurra** [comment]
Hi Clive, I am getting some errors when reproduce your branch. I made some comments in the code.
Another thing I would suggest to makes things easier:
1) Clone this repo, instead of working from your fork, otherwise I have to pull your branch from your personal repo, if I want to test that particular branch before merging.

Another thing, is that I managed to render Images in the list of cards, if you deploy my `branch: feature/cards-list`. That one is not throwing errors because I have a conditional rendering for those repos that don't follow the folder structure convention. Maybe is good if you try to incorporate your changes in this branch?

File: src/pages/JoseConnect.vue (hunk 1 above)
Comment (narration-sd) [reply src/pages/JoseConnect.vue]: Yes - I passed a tiny correction that fixed this oversight, but found you'd moved on for this branch, I thinking putting in that cards view you showed. 

Doesn't merge that easily, so I'll test it and bring it over, close the pull request...

**Comment by narration-sd** [comment]
Closed it -- think the merge problem occurred because I made the pull request against develop -- instead of against the feature/jose-queries branch. 

Anyway, we're both ahead now....

**Pull request closed without merging**